### PR TITLE
Initial tools

### PR DESCRIPTION
**Pull request by juandes**
This initial commit includes BBQ, and Tailer.

### CODE REVIEW

File: bbq/infer_schema.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```
Comment (ubunatic) [inline bbq/infer_schema.go:2]: Copied from google? Please add the source URL where you got this file.

**Review by ubunatic — COMMENTED** [review]
1. Did you check the applicability of the licenses? The Google code use Apache, we use BSD. Is that OK?
2. Do we need to add a License statement to each LOVOO-written file?

File: bbq/infer_schema.go (hunk 1 above)
Comment (juandes) [reply bbq/infer_schema.go:2]: That file is from Google :) from the BQ repo

**Comment by juandes** [comment]
> 1. Did you check the applicability of the licenses? The Google code use Apache, we use BSD. Is that OK?
> 2. Do we need to add a License statement to each LOVOO-written file?

1.  I believe it is ok. The Google code does not state that a copy of the license is needed.
2. I think that's not my call. However, none of our open sourced projects have a statement on the files.

**Review by ubunatic — APPROVED** [review]

**Review by frairon — APPROVED** [review]

**Pull request merged**